lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -36,7 +35,28 @@ function dosomething_campaign_preprocess_node(&$vars) {
   dosomething_campaign_preprocess_common_vars($vars, $wrapper);
 
   $pitch_path = 'node/' . $node->nid . '/pitch';
-  if ($pitch_path == $current_path || dosomething_campaign_is_pitch_page($node)) {
+
+  if (variable_get('rogue_collection', FALSE)) {
+    global $user;
+    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
```

Comment:
Let's use [`dosomething_user_get_northstar_id`](https://github.com/DoSomething/phoenix/blob/66f013d8df75117d8e7f392fe59036d752451799/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1270-L1284) to save a network request.